src/platform/core/src/device/shader/lcd_grid.glsl.hpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 fleroviux
+ *
+ * Licensed under GPLv3 or any later version.
+ * Refer to the included LICENSE file.
+ */
+
+#pragma once
+
+#include "device/shader/common.glsl.hpp"

```

Comment:
This header contains common vertex shader sources for shaders that do not perform anything special at the vertex stage. You're not using them here (assigning a custom shader source to `lcd_grid_vert`), so this include should be removed.